src/schema/v2/NotificationContent/index.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { ArtistType } from "../artist"
+import { ResolverContext } from "types/graphql"
+import { AlertType } from "../alerts"
+import { artworkConnection } from "../artwork"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { createPageCursors } from "../fields/pagination"
+import { connectionFromArray } from "graphql-relay"
+import { ArticleType } from "../article"
+import { ShowType } from "../show"
+
+const ArticleFeaturedArtistContentType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "ArticleFeaturedArtistNotificationContent",
+  fields: {
+    article: {
+      type: ArticleType,
+      resolve: ({ actor_ids }, _args, { articleLoader }) => {
+        if (!actor_ids) return null
+
+        const articleID = actor_ids[0]
+
+        return articleLoader(articleID)
+      },
+    },
+
+    artists: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(ArtistType))),
+      resolve: async ({ object_ids }, _args, { artistsLoader }) => {
+        if (!object_ids) return []
+
+        const { body } = await artistsLoader({ ids: object_ids })
+        return body ?? []
+      },
+    },
+  },
+})
+
+const ArtworkPublishedContentType = new GraphQLObjectType<any, ResolverContext>(
+  {
+    name: "ArtworkPublishedNotificationContent",
+    fields: {
+      artists: {
+        type: new GraphQLNonNull(
+          new GraphQLList(new GraphQLNonNull(ArtistType))
+        ),
+        resolve: async ({ actor_ids }, _args, { artistsLoader }) => {
+          if (!actor_ids) return []
+
+          const { body } = await artistsLoader({ ids: actor_ids })
+          return body ?? []
+        },
+      },
+
+      artworksConnection: {
+        type: artworkConnection.connectionType,
+        args: pageable(),
+        resolve: async ({ object_ids }, args, { artworksLoader }) => {
+          const { page, size } = convertConnectionArgsToGravityArgs(args)
+          const body = await artworksLoader({ ids: object_ids })
+          const totalCount = body.length
+
+          return {
+            totalCount,
+            pageCursors: createPageCursors({ page, size }, totalCount),
+            ...connectionFromArray(body, args),
+          }
```

Comment:
You can likely use that helper here, but this is a bit of a special case where we have all the `object_ids` ahead of time, and so we always request the _entire_ connection, and then paginate as a convenience to the client.

Thus, since the entire list is already fetched, it feels more idiomatic to use `connectionFromArray` (which helps you slice the already-fetched array), vs `connectionFromArraySlice` - so I'm fine with this form, they'll both be equivalent.